Swiftfin tvOS/Views/SearchView.swift

```diff
@@ -45,7 +45,6 @@ struct SearchView: View {
                 }
             }
         }
-        .ignoresSafeArea()
```

Comment:
We still need to ignore the safe area on the bottom, left, and right edges.

Re-add: `.ignoresSafeArea(edges: [.bottom, .horizontal])`

Reply:
Fair enough. I can add that back in. For my enlightenment why do we need it on the bottom and horizontal (left/right side)?

Reply:
We only need to ignore the safe area on the top, where the keyboard/tab bar are located. By not ignoring the horizontal and bottom sides, there will be padding from the safe area.

<details>
<summary>Not ignoring safe area sides</summary>

![Simulator Screenshot - Apple TV 4K (3rd generation) (at 1080p) - 2024-02-14 at 17 03 09](https://github.com/jellyfin/Swiftfin/assets/20747774/2e41996a-df7d-4d39-a854-61523be7e05b)

</details>

<details>
<summary>Ignoring safe area sides</summary>

![Simulator Screenshot - Apple TV 4K (3rd generation) (at 1080p) - 2024-02-14 at 17 02 52](https://github.com/jellyfin/Swiftfin/assets/20747774/3efdad97-de9a-4fef-94a9-ab368907b9ee)

</details>

Reply:
I see. I don't remember seeing that padding in my testing but agree with from your screenshots it makes sense to ignore. I have added it back in for the horizontal (left/right) and bottom. Thanks.